cypress/features/regression/test/flatTableIniFrame.feature

```diff
@@ -44,8 +48,8 @@ Feature: FlatTable component
   @positive
   Scenario Outline: Check value in <position> row
     # commented because of BDD default scenario Given - When - Then
-    # When I open "Flattable" component page
-    Then FlatTable <position> row contains proper inner content
+    # I open "test-flat-table" component page basic
```

Comment:
put the word `When` in front of `I open...`

Reply:
fixed